frontend/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.5.0",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
+    "@types/jest": "^24.9.1",
+    "@types/node": "^12.20.6",
+    "@types/react": "^16.14.5",
+    "@types/react-dom": "^16.9.11",
+    "@types/react-redux": "^7.1.16",
+    "plop": "^2.7.4",
```

Comment:
I think plop should be in devDependencies

Reply:
I was inspired that when I merged your branch, there were not devDependencies. I will change that.